css/style.css

```diff
@@ -2500,9 +2500,18 @@ footer .grid--4 {
     gap: 20px;
   }
 
+  #post-results.grid-view .article--container a {
```

Comment:
Rule already exists on line 2534

Reply:
I've left it as is to keep the list-view styles until Willie confirms whether we'll be using this layout on mobile. We may need to make some adjustments depending on his response.

Reply:
Sorry, Martin, I just saw your message on Asana. I'll make the correction on the css